src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -699,7 +698,22 @@ private void RobustDelete(string path)
             }
         }
 
+        /// <summary>
+        /// Run msbuild.exe within corerun.exe
+        /// It concatenates the given msbuild path to the given parameters and sends it to corerun.
+        /// </summary>
+        private string ExecMSBuild(string pathToMsBuildExe, string msbuildParameters, bool expectSuccess = true)
+        {
 #if FEATURE_RUN_EXE_IN_TESTS
```

Comment:
Yes, I agree.  I was saying that the feature flag `RUN_EXE_IN_TESTS` is now awkward:  it's really controlling whether to use corerun or not.  We need to make that decision but this seems like an inelegant way.